apps/settings/style/downloads.css

```diff
@@ -235,6 +235,11 @@
 }
 
 /* RTL rules */
+html:-moz-dir(rtl) #downloadList aside.download-status {
+  right: 0.2rem;
```

Comment:
It's fine to set `right` to 0.2rem, but we should also flip the margin of `#downloadList aside` in RTL mode (`margin: 1.75rem 0 0 0.5rem`).